sponge/__init__.py

```diff
@@ -1,7 +1,30 @@
 from flask import Flask
```

Comment:
This seems to cause issues with virtualenv, such that it can't install depends until after in looks at the `sponge` package.